release/release_tests.yaml

```diff
@@ -2579,6 +2638,7 @@
   team: core
   frequency: nightly
   working_dir: microbenchmark
+  env: prod
```

Comment:
is it a mistake ?

Reply:
No this is deliberate - there is a `microbenchmark_staging` test and `microbenchmark` should be on prod. This is to make sure this remains after we eventually make a switch to default to staging